src/app/page/Missions/Missions.jsx

```diff
@@ -39,9 +39,12 @@ const MissionsPage = ({ user, history, firebase, ...rest }) => {
       User.assignAsVolunteer(firestore, missionId, user.uid);
     }
   }
-
-  if (!missions) {
-    return <div> isloading...</div>;
+  if (missions === undefined) {
+    return (
+      <Grid container justify="center" alignItems="center" style={{ minHeight: "100vh" }}>
```

Comment:
We can refactor the spinner + grids into a component so that we can reuse it at other places!

Reply:
Any thoughts on this? @malignantz